src/protocol/data/resp/request.h

```diff
@@ -75,17 +75,22 @@ struct command {
 extern struct command command_table[REQ_SENTINEL];
 
 struct request {
-    STAILQ_ENTRY(request)   next;     /* allow request pooling/chaining */
+    STAILQ_ENTRY(request)   next;   /* allow request pooling/chaining */
     bool                    free;
 
-    bool                    noreply;  /* skip response */
-    bool                    serror;   /* server error */
-    bool                    cerror;   /* client error */
+    bool                    noreply;/* skip response */
+    bool                    serror; /* server error */
+    bool                    cerror; /* client error */
 
     cmd_type_e              type;
-    struct array            *token;    /* member type: `struct element' */
+    struct array            *token; /* member type: `struct element' */
+
+    /* global attributes */
+    int64_t                ttl;     /* -1 means it is not provided */
```

Comment:
given the INT64_MIN figure given in the reset function, is that what you mean here? the comment says -1

Reply:
Yes- good catch. I originally wanted to use `-1` for not provided, then decided I may want to reserve `-1` (or other small negatives) for some particular meaning in the future, such as infinite or invalid TTL.